app/views/partials/foyer/individu-form.html

```diff
@@ -184,21 +184,24 @@
     </div>
   </div>
 
-  <div class="form-group from-group-lg" ng-if="captureTauxInvalidite()">
+  <div class="form-group from-group-lg" ng-if="capturetauxIncapacite()">
     <label class="control-label col-sm-4" for="eligibiliteAss">
-      Quel est {{ isDemandeur() ? 'votre' : 'son' }} taux d'invalidité ?
+      Quel est {{ isDemandeur() ? 'votre' : 'son' }} taux d'incapacité ?
+      <div class="small">
```

Comment:
Is it a `small` or is it a `help-block` or a `help-col`? Where else do we use `small`?